database/methods/update/reviewsummary.js

```diff
@@ -0,0 +1,65 @@
+const ReviewModel = require('../../models/reviews.js');
+const ReviewSummary = require('../../models/reviewsummary.js');
+
+const updateRating = async (filter, newData) => {
```

Comment:
Why do you need async here? You don't seem to be awaiting for anything within this function scope. Noticing the same thing for most functions in this file